lualibs/go/ngx/loader.go

```diff
@@ -19,6 +20,29 @@ func LoadNgx(l *lua.LState) {
 	mt := l.NewTable()
 	l.SetFuncs(mt, map[string]lua.LGFunction{"__tostring": NullString})
 	l.SetField(ngx, "null", d)
+
+	// fetch gluare.find and gluare.match functions
+	// see https://github.com/yuin/gluare/blob/master/gluare.go
```

Comment:
Could we add a blurb here talking about the differences between `ngx.re.find` and `go.re2.find`? Since ngx.re.find uses PCRE and go.re2.find uses RE2 we know there is a difference and we should let futureselves know when we look at the code again.

Reply:
Documented LoadNgx as well as the overrides of find and match.